graspologic/align/seedless_procrustes.py

```diff
@@ -270,7 +272,9 @@ def _optimal_transport(self, X, Y, Q):
         cost_matrix = (
             np.linalg.norm((X @ Q).reshape(n, 1, d) - Y.reshape(1, m, d), axis=2) ** 2
         )
-        P = ot.sinkhorn(
+        # Note: the type info on ot.sinkhorn can support a return type of Tuple[np.ndarray, Dict[str, Any]], but only
+        # if this function is called with log=True, which we aren't doing.
+        P: np.ndarray = ot.sinkhorn(
```

Comment:
This is just. a personal preference but it helps with a cognitive cost when looking at things later - if I call another library that doesn't use type hinting, I can identify the type of value it returns in my own code after I've done the sleuthing once when reading through their own code.  Then when I come back to it later, I don't have to do all that sleuthing over again just to track down what the hell it's supposed to be.